docs/data/guides/telemetry/telemetry.md

```diff
@@ -76,9 +76,7 @@ Note: You can use `muiXTelemetrySettings.enableDebug();` to log telemetry data t
 You can set the `__MUI_X_TELEMETRY_DISABLED__` flag in your application to `false` to enable telemetry:
 
 ```js
-import { ponyfillGlobal } from '@mui/utils';
-
-ponyfillGlobal.__MUI_X_TELEMETRY_DISABLED__ = false;
+globalThis.__MUI_X_TELEMETRY_DISABLED__ = false;
```

Comment:
I have removed the `as any` from here as it implies that users are on `TS`, which is not a given.